examples/features/standard/broker-plugin/src/test/java/org/apache/activemq/artemis/jms/example/OpenTracingBrokerPluginTest.java

```diff
@@ -18,51 +18,46 @@
 
 import java.util.List;
 
+import io.opentelemetry.api.GlobalOpenTelemetry;
 import io.opentelemetry.api.trace.SpanKind;
 import io.opentelemetry.sdk.testing.exporter.InMemorySpanExporter;
 import io.opentelemetry.sdk.trace.data.SpanData;
+import org.apache.activemq.artemis.api.core.ActiveMQException;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
 
 import static io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions.assertThat;
 import static org.junit.Assert.assertNotNull;
 
+@RunWith(MockitoJUnitRunner.class)
 public class OpenTracingBrokerPluginTest {
 
    private final InMemorySpanExporter exporter = InMemorySpanExporter.create();
-   //private SdkTracerProvider sdkTracerProvider;
-   //private Tracer tracer;
 
-   /**
-    * @Before public void setUp() {
-    * sdkTracerProvider = SdkTracerProvider.builder()
-    * .addSpanProcessor(SimpleSpanProcessor.create(exporter))
-    * .build();
-    * tracer = OpenTelemetrySdk.builder()
-    * .setTracerProvider(sdkTracerProvider)
-    * .buildAndRegisterGlobal()
-    * .getTracer("org.apache.activemq.artemis.jms.example.OpenTracingBrokerPlugin");
-    * }
-    **/
+   @Mock
+   OpenTracingBrokerPlugin plugin = new OpenTracingBrokerPlugin();
 
    @Test
-   public void testSpans() {
+   public void assertInitOpenTracing() {
```

Comment:
@franz1981 running this test shows me and error 
```
Failed to read or configure the org.jboss.logmanager.LogManager
java.lang.NoClassDefFoundError: org/apache/activemq/artemis/shaded/org/jboss/logmanager/ExtHandler
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1017)
	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:174)
	at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:800)
	at java.base/jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:698)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:621)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:579)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:398)
	at org.jboss.logmanager.config.AbstractPropertyConfiguration.<init>(AbstractPropertyConfiguration.java:72)
	at org.jboss.logmanager.config.HandlerConfigurationImpl.<init>(HandlerConfigurationImpl.java:51)
	at org.jboss.logmanager.config.LogContextConfigurationImpl.addHandlerConfiguration(LogContextConfigurationImpl.java:143)
	at org.jboss.logmanager.PropertyConfigurator.configureHandler(PropertyConfigurator.java:673)
	at org.jboss.logmanager.PropertyConfigurator.configureLogger(PropertyConfigurator.java:583)
	at org.jboss.logmanager.PropertyConfigurator.configure(PropertyConfigurator.java:521)
	at org.jboss.logmanager.PropertyConfigurator.configure(PropertyConfigurator.java:97)
	at org.jboss.logmanager.LogManager.readConfiguration(LogManager.java:170)
	at org.jboss.logmanager.LogManager.readConfiguration(LogManager.java:132)
	at java.logging/java.util.logging.LogManager.readPrimordialConfiguration(LogManager.java:445)
	at java.logging/java.util.logging.LogManager$2.run(LogManager.java:394)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.logging/java.util.logging.LogManager.ensureLogManagerInitialized(LogManager.java:382)
	at java.logging/java.util.logging.LogManager.getLogManager(LogManager.java:430)
	at java.logging/java.util.logging.Logger.demandLogger(Logger.java:648)
	at java.logging/java.util.logging.Logger.getLogger(Logger.java:717)
	at java.logging/java.util.logging.Logger.getLogger(Logger.java:701)
	at io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdkBuilder.<clinit>(AutoConfiguredOpenTelemetrySdkBuilder.java:43)
	at io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdk.builder(AutoConfiguredOpenTelemetrySdk.java:37)
	at io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdk.initialize(AutoConfiguredOpenTelemetrySdk.java:29)
	at org.apache.activemq.artemis.jms.example.OpenTracingBrokerPlugin.initOpenTracing(OpenTracingBrokerPlugin.java:50)
	at org.apache.activemq.artemis.jms.example.OpenTracingBrokerPlugin.<clinit>(OpenTracingBrokerPlugin.java:37)
	at org.apache.activemq.artemis.jms.example.OpenTracingBrokerPluginTest.<init>(OpenTracingBrokerPluginTest.java:39)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at org.junit.runners.BlockJUnit4ClassRunner.createTest(BlockJUnit4ClassRunner.java:250)
	at org.junit.runners.BlockJUnit4ClassRunner.createTest(BlockJUnit4ClassRunner.java:260)
	at org.junit.runners.BlockJUnit4ClassRunner$2.runReflectiveCall(BlockJUnit4ClassRunner.java:309)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
Caused by: java.lang.ClassNotFoundException: org.apache.activemq.artemis.shaded.org.jboss.logmanager.ExtHandler
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	... 61 more

Process finished with exit code 0
```

Reply:
i still face the same error and am trying to solve it.

Reply:
@franz1981 I have made some changes in the test logic but the error i shared above is blocking still blocking me.